glocal_exploration/src/planning/global/skeleton/skeleton_a_star.cpp

```diff
@@ -202,10 +202,10 @@ bool SkeletonAStar::getPathBetweenVertices(
 
     const voxblox::Point t_odom_current_vertex =
         current_submap.getPose() * current_vertex.point;
+    const FloatingPoint h_score = (goal_point - t_odom_current_vertex).norm();
     g_score_map[current_vertex_id] =
         (t_odom_current_vertex - start_point).norm();
-    f_score_map[current_vertex_id] =
-        (goal_point - t_odom_current_vertex).norm();
+    f_score_map[current_vertex_id] = h_score;
```

Comment:
Is this intentionally called h_score and then assigned to the f_score_map? I'm not too familiar with the terminology but might lead to confusions.

Reply:
Strictly speaking, the update equation here should be [`f[neighbor] = g[neighbor] + h[neighbor]`](https://en.wikipedia.org/wiki/A*_search_algorithm#Pseudocode). Since the `current_vertex_id`s in the line above are the starting vertices, they'll be expanded in pretty much any search so it won't affect the result. But I agree it's confusing.
In general, this method grew organically and got quite messy. I'll do some refactoring.